pilot/pkg/model/context.go

```diff
@@ -784,6 +784,11 @@ func (node *Proxy) SetSidecarScope(ps *PushContext) {
 	node.PrevSidecarScope = sidecarScope
 }
 
+func (node *Proxy) ServiceInSidecarScope(service *Service) bool {
```

Comment:
Moving this to sidecar makes sense

Reply:
can you remove now that its unused? Makes the PR review eaieeasier